install_sct

```diff
@@ -757,3 +760,5 @@ else
     die "Installation validation Failed!"
   fi
 fi
+
```

Comment:
One last thought before we merge -- would there be a easy way to get `set -x` functionality in the install log, but not in the stdout? (I'm guessing that would require a more complicated solution than `tee`, aha)

I ask because often when debugging a user's issue I feel like I want them to re-run with verbose mode on, but that can be a bit of a hassle. It would be nice if we could have access to that by default.

Reply:
Maybe with `| egrep -v '^\+ '`? Maybe with, uh, there's some syntax like `exec 4<>/path/to/fifo` that opens temporary extra streams you can use besides stdout and stderr. But either seems fragile.

Reply:
Probably not a huge deal, then. :stuck_out_tongue_closed_eyes: